knowledge-base/cannot-show-report-reportviewer.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Cannot Show Reports in the ReportViewer
+page_title: Unable to Show Reports in the ReportViewer
+description: "Learn how to programmatically specify a report in a separate class library in the Telerik ReportViewer."
+slug: cannot-show-report-reportviewer
+previous_url: /troubleshooting/design-time
+tags: telerik, reporting, design, time, cannot, show, report, reportviewer
+published: True
+component: reportviewer
+type: troubleshooting
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+			<td>Progress® Telerik® ReportViewer</td>
+		</tr>
+		<tr>
+			<td>Version</td>
+			<td>Q2 2012 and newer</td>
+		</tr>
+	        <tr>
+			<td>Report Viewers</td>
+			<td>WinForms, WPF, ASP.NET</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+I want to show my report in a ReportViewer control, but when I click the arrow in the **ReportSource** property from the property grid, it does not show available reports. What is wrong?
+
+## Cause
+
+....
+
+## Solution  
+
+Follow the Telerik Reporting [best practices](66CD7D60-7708-42D5-8BB4-506676E8679E) and specify the report in a separate class library that is referenced in the application or website. Check if the class library containing the report is referenced in your application or website and that you have rebuilt the project.
```

Comment:
the cited documentation article is redirected to "https://docs.telerik.com/reporting/quickstart/overview"

Reply:
I've added the slug, thanks. 
Do we plan on publishing the quickstart folder and if so, where? 
Also, please fill in the dots under Cause.